### PR TITLE
🌱 update deps to v0.14

### PR DESCRIPTION
**Pull request by qiujian16**
[comment]: # ( Copyright Contributors to the Open Cluster Management project )

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by qiujian16** [comment]
@dhaiducek @mikeshng would you check if anything needs to be updated for policy and app addon?

File: pkg/version/version.go [hunk 1]
```diff
@@ -149,6 +128,17 @@ func GetVersionBundle(version string) (VersionBundle, error) {
...
 	}
 
+	versionBundleList["0.14.0"] = VersionBundle{
+		Registration:             "0.14.0",
+		Placement:                "0.14.0",
+		Work:                     "0.14.0",
+		Operator:                 "0.14.0",
+		AddonManager:             "0.14.0",
+		AppAddon:                 "0.14.0",
+		PolicyAddon:              "0.14.0",
```
Comment (dhaiducek) [inline pkg/version/version.go:138]: Why are there no `v` prefixes here? Is that handled elsewhere?

**Review by dhaiducek — CHANGES_REQUESTED** [review]
I've just released v0.14.0 for Policy, so we're good there. I have one question about the lack of `v` prefix in this PR.

**Review by mikeshng — APPROVED** [review]
/lgtm

Good catch by Dale. Otherwise, all good to go.

File: pkg/version/version.go (hunk 1 above)
Comment (qiujian16) [reply pkg/version/version.go]: updated

**Review by mikeshng — APPROVED** [review]
/lgtm

**Comment by dhaiducek** [comment]
Also, the default version here is still set to `v1.13.3`. That will need to be updated before release.

**Pull request merged**